### PR TITLE
[FIX] base, tests: running phantomjs tests on windows machine

### PR DESCRIPTION
**Pull request by suvit**

### CODE REVIEW

File: openerp/tests/common.py [hunk 1]
```diff
@@ -286,7 +286,11 @@ def phantom_poll(self, phantom, timeout):
                 err, _ = e.args
                 if err == errno.EINTR:
                     continue
-                raise
+                if err == 10038 and os.name == 'nt':
```
Comment (mart-e) [inline openerp/tests/common.py]: isn't there an `errno` constant for that instead of hardcoding the number?

Comment (suvit) [inline openerp/tests/common.py]: may be, I look

Comment (suvit) [inline openerp/tests/common.py]: Yes, its errno.ENOTSOCK

**Comment by mart-e** [comment]
Thanks for the update. Could you give me a way to reproduce and source on the "select.select dont work with file descriptors on windows"?
Thanks

**Comment by suvit** [comment]
https://docs.python.org/2/library/select.html#select.select
See 'note' section

**Comment by C3POdoo** [comment]
Hello,

This is an automatic message.

Thank you for your contribution but the version 8.0 is no longer supported. We only support the last 3 stable versions so no longer accepts patches into this branch.
We apology if we could not look at your request in time.

If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

Regards,

**Pull request closed without merging**